packages/core/src/lib/TwaGenerator.ts

```diff
@@ -226,10 +257,25 @@ export class TwaGenerator {
     await fs.promises.writeFile(webManifestFileName, JSON.stringify(webManifestJson));
   }
 
+  private async monochromeFilter(icon: Icon, twaManifest: TwaManifest): Promise<Icon> {
```

Comment:
Maybe we could extract this to a `JimpHelper` or `ImageHelper` and change the signature to take the image data and a background colour and make it more testable.